src/serializer/modules.js

```diff
@@ -196,7 +202,7 @@ class ModuleTracer extends Tracer {
           invariant(popped === moduleIdValue);
           let message = "";
           invariant(!(result instanceof IntrospectionThrowCompletion));
-          if (result instanceof ThrowCompletion) (" threw an error");
+          if (result instanceof ThrowCompletion) " threw an error";
```

Comment:
While you are at it, you could fix this too... Should be `message = " threw an error";`